inputremapper/injection/mapping_handlers/abs_to_rel_handler.py

```diff
@@ -98,7 +98,7 @@ async def _run_wheel_output(self, codes: Tuple[int, int]) -> None:
             self._write(EV_REL, codes[i], value)
 
         time_taken = time.time() - start
-        await asyncio.sleep(max(0.0, (1 / self.mapping.rel_wheel_rate) - time_taken))
+        await asyncio.sleep(max(0.0, (1 / self.mapping.rel_rate) - time_taken))
```

Comment:
That is interesting, I have two mice which I just tested:

Razer Naga Trinity:
Rate of 1000Hz for the cursor and the wheel is as fast as I can spin it. It also tops out at 1000Hz.

Logitech G602:
same as the Razer but with a rate of 500Hz

So I guess it is different for each mouse, I originally chose 100Hz because it matches the refresh rate of my monitor and the mouse moves smooth. So maybe we choose 144Hz?

Also I don't think anyone will notice a difference if wheel and mouse events are at the same rate or not.

Reply:
For your trinity, since the frequency is about 10 times higher, is the value 10 times lower then so that the speed stays the same?

Reply:
most of the time it is 1-3. If I move it fast it gets up to 15.

Reply:
ok. the 30 I set for `rel_speed` are based on the 125hz rate of my mouse.

we could define `rel_speed` as "relative movement per second", the injected value would be `rel_speed / rel_rate`

Reply:
that makes sense. we should do that

Reply:
ok, yeah. Do you want to include that change in this PR? Otherwise I'll just do it on the rel2rel PR

Reply:
just do it in your PR and merge this.

Reply:
rel2rel example:
- input every 0.1s (rate of 10 events/s), value of 200
- input speed is 2000, because in 1 second a value of 2000 acumulates
- input_rel_speed is a const defined as 4000 px/s, how fast mice usually move
- the fractional-speed is 0.5
- throw 0.5 into the Transformation, and get 0.5 because the expo is 0
- rel_speed is 5000
- inject 2500 therefore per second, making it a bit faster
- divide 2500 by the rate of 10 to inject a value of 250 each time input occurs

input_rate cancels out:
- input-value / input_rel_speed * rel_speed
- = input-value * rel_speed / input_rel_speed
- = input-value * (rel_speed / input_rel_speed)

`rel_speed / input_rel_speed` would be a fractional value, just like the gain of the
Transformation

so instead of both rel_speed and input_rel_speed, the gain could be used to achieve the
exact same thing.

however
- we need a base value for the output of abs2rel, so rel_speed is still required. We
also have the gain though..
- in rel2rel, the min/max value of the Transformation makes a difference for the output
if an expo is set. If the max is too large, and expo is positive, the output value will
always be smaller. Making the expo a setting that not only controls the curvature, but
also the gain at the same time, which is not desired. So we would need input_rel_speed
here, and don't need rel_speed at all.

that means rel_speed is only required in abs2rel. I would therefore turn rel_speed into
abs_to_rel_speed.